plugins/badges-backend/src/service/router.ts

```diff
@@ -48,10 +51,14 @@ export async function createRouter(
 
```

Comment:
I think it makes sense here above the route handlers to just put in a nested function, something like the below that can be called, instead of repeating the same stuff over and over. I.e. it would make sense to have such a little function to copy paste at the top of createRouter functions in the files below, and reducing each `(req, res)` handler function to getting it with a oneliner.

```ts
async function getRequestToken(request: Request): Promise<string | undefined> {
  if (options.identity) {
    const { token ) = await options.identity.getIdentity({ request });
    return token;
  }
  return getBearerTokenFromAuthorizationHeader(request.headers.authorization);
}
```